snippets/language-ruby.cson

```diff
@@ -114,6 +114,9 @@
   'def … end':
     'prefix': 'def'
     'body': 'def ${1:method_name}\n\t$0\nend'
+  'initialize':
+    'prefix': 'init'
+    'body': 'def initialize${1:(${2:arguments, …})}\n\t$0\nend'
```

Comment:
@exalted What do you think about using `${2:argument}` instead of `${2:arguments, …}` ? I think I've seen that pattern being used in some other snippets (e.g. [here](https://github.com/atom/language-javascript/blob/49fe9a49deb9fc9ede5f1a94848a3d5f5dea8772/snippets/language-javascript.cson#L35-L37)).